### ISSUE TITLE
FR: Truck config block - Phase 2

### ISSUE THREAD

**Issue by shirin27** [Netcentric/vg-macktrucks-com-rd#33]
Add the tabbed column where the image changes

https://www.figma.com/file/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?node-id=2243%3A159073&mode=dev

**Comment by WendyKruger** [comment]
> Add the tabbed column where the image changes
> 
> https://www.figma.com/file/oNwe2jHIICPMIe551EY62l/Mack-Reskin---Full-Project?node-id=2243%3A159073&mode=dev

Can you please explain what exactly this means?

**Comment by BeckyMedlin** [comment]
@shirin27 Need clarity on why this is needed.

**Comment by shirin27** [comment]
@BeckyMedlin this ticket is a split from #32 , as the scope was too big when we initially refined the ticket.
I will clarify the exact scope of each ticket in a refinement session with the team, has the Truck configuration block become a priority? Should I bring it to refinement this week?

**Comment by BeckyMedlin** [comment]
@shirin27 No, it's still a priority 3. I asked @WendyKruger yesterday if the config blocks needed to be included with the Configurator work and she said no.

**Comment by shirin27** [comment]
Kamino cloned this issue to [hlxsites/vg-macktrucks-com](https://github.com/hlxsites/vg-macktrucks-com/issues/610)
